src/scripts/main.js

```diff
@@ -1 +1,66 @@
 'use strict';
+
+function showMessage(message, isError = false) {
+  const messageDiv = document.createElement('div');
+
+  messageDiv.classList.add(isError ? 'error' : 'success');
+  messageDiv.setAttribute('data-qa', 'notification');
+  messageDiv.textContent = message;
+  document.body.appendChild(messageDiv);
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+firstPromise
+  .then((message) => {
+    showMessage(message);
+  })
+  .catch((error) => {
+    showMessage(error.message, true);
+  });
+
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('click', (e) => {
+    if (e.button === 0 || e.button === 2) {
```

Comment:
The condition `e.button === 0 || e.button === 2` checks for left or right mouse button clicks. Ensure this is the intended behavior, as it resolves the promise on any of these clicks.